json-modules/json-conversion/src/test/java/com/baeldung/jsonnodetojsonobject/JsonNodeToJsonObjectUnitTest.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.jsonnodetojsonobject;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class JsonNodeToJsonObjectUnitTest {
+    public static String jsonString = "{\"name\": \"John\", \"gender\": \"male\", \"company\": \"Baeldung\", \"isEmployee\": true, \"age\": 30}";

```

Comment:
Can we add a line between the string and the class definition?

Reply:
Done